.github/workflows/training_workflow.yml

```diff
@@ -6,69 +6,78 @@ on:
     branches:
       - master
 
+permissions:
+  id-token: write
+  contents: read
+
 jobs:
-  build_and_push:
-    runs-on: ubuntu-latest
+  # build_and_push:
+  #   runs-on: ubuntu-latest
 
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v2
+  #   steps:
+  #     - name: Checkout repository
+  #       uses: actions/checkout@v4
       
-      #- name: Set MLFLOW_TRACKING_URI environment variable in Dockerfile
-      #  run: |
-      #    sed -i 's|ENV MLFLOW_TRACKING_URI=.*|ENV MLFLOW_TRACKING_URI=s3://${{ secrets.AWS_S3_BUCKET_NAME}}/models/mlflow-runs|' \
-      #    ./experiment/train/Dockerfile
+  #     #- name: Set MLFLOW_TRACKING_URI environment variable in Dockerfile
+  #     #  run: |
+  #     #    sed -i 's|ENV MLFLOW_TRACKING_URI=.*|ENV MLFLOW_TRACKING_URI=s3://${{ secrets.AWS_S3_BUCKET_NAME}}/models/mlflow-runs|' \
+  #     #    ./experiment/train/Dockerfile
 
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
+  #     - name: Set up QEMU
```

Comment:
Do we need to keep the commented job ? True question, maybe we can uncomment it or move it to a separate workflow ?